cli/src/commands/workspace.rs

```diff
@@ -279,6 +280,22 @@ fn cmd_workspace_root(
     Ok(())
 }
 
+/// Loads workspace that will diverge from the last working-copy operation.
+fn for_stale_working_copy(
+    command: &CommandHelper,
+    ui: &mut Ui,
+) -> Result<WorkspaceCommandHelper, CommandError> {
```

Comment:
nit: reorder the arguments to `(ui, command)` like the other functions

Reply:
Done